pastepwn/analyzers/azuresubscriptionkeyanalyzer.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from .regexanalyzer import RegexAnalyzer
+
+
+class AzureSubscriptionKeyAnalyzer(RegexAnalyzer):
+    """
+    Analyzer to match Azure subscription key via regex
+
+    Keys are 32 character alphanumeric (lower case)
+    """
+    name = "AzureSubscriptionKeyAnalyzer"
+
+    def __init__(self, actions):
+        # https://docs.microsoft.com/en-us/azure/api-management/api-management-subscriptions
+        regex = r"^[a-f0-9]{32}$"
```

Comment:
Same as in the other PR - could you please replace the `^` and `$` with word boundaries (`\b`)? Otherwise we can't match those api keys in the middle of a sentence.